api/proto/teleport/legacy/types/types.proto

```diff
@@ -5263,6 +5263,42 @@ message OktaAssignmentActionTargetV1 {
   string id = 2 [(gogoproto.jsontag) = "id"];
 }
 
+// IntegrationV1 represents a connection between Teleport and some other 3rd party system.
+// This connection allows API access to that service from Teleport.
+// Each Integration instance must have a SubKind defined which identifies the external system.
+message IntegrationV1 {
+  option (gogoproto.goproto_stringer) = false;
+  option (gogoproto.stringer) = false;
+
+  // Header is the resource header.
+  ResourceHeader header = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.jsontag) = "",
+    (gogoproto.embed) = true
+  ];
+
+  // Spec is an Integration specification.
+  IntegrationSpecV1 Spec = 2 [
```

Comment:
i think generally we prefer lowercase snaking? (but in this file i see a mixture of both)

Reply:
I'll convert to `Header` and `Spec`
Seems the most common pattern for the top level fields of a resource.